android/app/src/main/java/org/fossasia/openevent/adapters/SpeakersListAdapter.java

```diff
@@ -76,6 +76,7 @@ protected void publishResults(CharSequence constraint, FilterResults results) {
     public SpeakersListAdapter(List<Speaker> speakers, Activity activity) {
         super(speakers);
         this.activity = activity;
+        this.picassoCache = new PicassoCache(activity.getApplicationContext());
```

Comment:
I went through [guide1](https://medium.com/freenet-engineering/memory-leaks-in-android-identify-treat-and-avoid-d0b1233acc8),  [guide2](https://possiblemobile.com/2013/06/context/) for memory leak in android. As we are not making **static** instance of context it will not leak memory. I checked it in HPROF Viewer and Analyzer in android studio.

Tell me if i am wrong at some point and suggest better way to do this.